### DISCUSSION TITLE
Disable Normalization and Normalizing the particular entity?

### DISCUSSION THREAD

**Discussion by yogeshwaran-ramesh** [urql-graphql/urql#2552] [Q&A] [answered]
Hi there,
1. Is there a way to disable normalization and to store denormalized data?
2. Any way normalize only one particular entity?

**Comment by kitten** [comment, marked as answer]
Can you be more specific please?
Do you mean for a specific entity? All that that'd require is an unkeyed entity, i.e an embedded one 
This is documented here: https://formidable.com/open-source/urql/docs/graphcache/normalized-caching/#custom-keys-and-non-keyable-entities

**Reply by yogeshwaran-ramesh** [reply]
yes sure,
I have 2 questions,
1. For example,  I need to disable normalization and store denormalized data in the cache. is it possible?
2. `{
  todo {
    id
    title
    author {
      id
      name
    }
  }
}`
For the above one, what if I need to normalize only todo based on id, I don't want to normalize author ?
